atomic-swap/build.gradle

```diff
@@ -0,0 +1,71 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'kotlin-android-extensions'
+
+}
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        minSdkVersion 22
+        targetSdkVersion 29
```

Comment:
Please ensure you are targetting [API level 31](https://developer.android.com/google/play/requirements/target-sdk). In this state, your app will not be able to be published on the play store.

Reply:
Have you also tested your app after this, it can lead to quite some issues?

Reply:
Yes, we've tested it together with @rahimklaber and it's working.

Reply:
This still shows 29 for me.